src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -144,7 +144,7 @@ public async Task<bool> IsEnabledAsync(string feature)
         /// Checks whether a given feature is enabled.
         /// </summary>
         /// <param name="feature">The name of the feature to check.</param>
-        /// <param name="appContext">A context providing information that can be used to evaluate whether a feature should be on or off.</param>
+        /// <param name="appContext">A context that provides information that can be used to evaluate whether a feature should be on or off.</param>
```

Comment:
No need for this change. This causes nested "that" clauses.

Reply:
I will change to "A context that provides information to evaluate whether a feature should be on or off."

Reply:
sure, it works.